app/scripts/auth.js

```diff
@@ -0,0 +1,72 @@
+import $ from 'jquery';
+const firebase = require('firebase');
+
+require('./config.js');
+const Form = require('./form.js');
+
+class Auth {
+
+    constructor () {
+        $('.js-auth-form').submit( (e) => {
+
+            var $this = $(e.currentTarget);
```

Comment:
The name `$this` sounds weird for me... Usually I try to associate this name with the `this` of the language.

What do you think about change this name to just `target` or something like that?